### PR TITLE
Update solution_architecture.md - UA Review

### PR DESCRIPTION
**Pull request by SabineLoss**
Some comments for the reviewers:
- German term is "Corona-positiv getestete Personen"; here, we mostly use "positively tested individuals". Should we add "Corona"?
- line 36 @OlMue : Please double-check
- line 57: What is "conventional"?
- line 69: Add link to Scoping doc
- line 165: @OlMue , @SebastianWolf-SAP : Can you please check this sentence? I'm not sure I fully understand it.

The capitalization of terms like diagnosis key, temporary exposure key, etc. is not consistent. I didn't have the time today to harmonize it, but as it is not relevant for the German translation, we can do this at a later stage.

### CODE REVIEW

**Comment by SebastianWolf-SAP** [comment]
Just saw that we have conflicts. They should be resolved before merging - that would also make it possible to fix the typo. ;)

**Comment by SabineLoss** [comment]
> Just saw that we have conflicts. They should be resolved before merging - that would also make it possible to fix the typo. ;)

When fixing this typo I found another typo. Both are fixed now :-)

**Comment by SabineLoss** [comment]
> Just saw that we have conflicts. They should be resolved before merging - that would also make it possible to fix the typo. ;)

Conflict is resolved as well.

File: solution_architecture.md [hunk 1]
```diff
@@ -233,24 +235,24 @@ The backend will be made available through the [Open Telekom Cloud (OTC)](https:
 
 For the operation of the backend, several bandwidth estimations need to be taken. As a high adoption rate of the app is an important requirement, we are considering up to 60 million active users.
```
Comment (OlMue) [inline solution_architecture.md:236]: backend -> back end

File: solution_architecture.md [hunk 2]
```diff
@@ -233,24 +235,24 @@ The backend will be made available through the [Open Telekom Cloud (OTC)](https:
...
 
-### Bandwidth estimations
+### Bandwidth Estimations
 
 While each set of 14 diagnosis keys only has the small size of 392 Byte, all newly submitted diagnosis keys of a time period need to be downloaded by all mobile phones having the application installed. When considering 2000 new cases for a day, a transmission overhead (through headers, handshakes, failed downloads, etc.) of 100% and 30 million clients, this adds up to approximately 1.5MB per client, i.e. **42.8TB** overall traffic. If a day is split into 24 chucks (one per hour), this results in a total number of **720 million requests per day**. If the requests are evenly spread throughout the corresponding hour, approximately **8,500 session requests** per second need to be handled. Detailed descriptions of the connections can be found in the chapter ["Data transfer and data processing"](#data-transfer-and-data-processing). Those number exclude possible interoperability with other countries.
```
Comment (OlMue) [inline solution_architecture.md:240]: - 392 Byte -> 392 bytes
- **42.8TB** overall traffic -> **42.8TB** of overall traffic

File: solution_architecture.md [hunk 3]
```diff
@@ -233,24 +235,24 @@ The backend will be made available through the [Open Telekom Cloud (OTC)](https:
...
 
 If the backend calls from the mobile applications cannot be spread as evenly as we expect, we might experience peaks. To achieve an even spread (and to be able to handle a peak), we will employ a Content Delivery Network (CDN) by T-Systems to make the individual aggregated files available. According to an evaluation with T-Systems, the estimated throughput and request number can be handled by their infrastructure.
```
Comment (OlMue) [inline solution_architecture.md:242]: backend -> back end

File: solution_architecture.md [hunk 4]
```diff
@@ -233,24 +235,24 @@ The backend will be made available through the [Open Telekom Cloud (OTC)](https:
...
+In *Figure 13*, this is visualized while focusing on the captured Rolling Proximity Identifiers by only a single device. We are assuming that devices broadcast their own RPI every 250ms and use listening windows with a length of two seconds, five minutes apart. There are five other active devices – each representing a different kind of possible exposure. In the example, devices 3 and 4 go completely unnoticed, while a close proximity with the user of device 2 cannot be detected. In contrast to that very brief, but close connection with the user of device 5 (e.g. only brushing the other person in the supermarket) is noticed and logged accordingly. The duration and interval of scanning needs to be balanced by Apple and Google against battery life, as more frequent scanning consumes more energy.
```
Comment (OlMue) [inline solution_architecture.md:256]: We are assuming -> We assume

File: solution_architecture.md [hunk 5]
```diff
@@ -233,24 +235,24 @@ The backend will be made available through the [Open Telekom Cloud (OTC)](https:
...
 
 It must be noted, that some of the encounters described above are corner cases. While especially the cases with a very short proximity time cannot be detected due to technical limitations, the framework will be able to detect longer exposures. As only exposures of longer duration within a certain proximity range are considered relevant for the intended purpose of this app, most of them will be covered.
```
Comment (OlMue) [inline solution_architecture.md:258]: It must be noted, that -> It must be noted that

File: solution_architecture.md [hunk 6]
```diff
@@ -78,104 +78,106 @@ The flow for using the app is as follows, referencing the steps from *Figure 2*:
...
+From a privacy protection perspective, sending push notifications via Apple’s or Google’s push service is not acceptable in this scenario. Even though no specific test results are included in the notifications, the message itself signals that the user has taken a SARS-CoV-2 test. Thus, polling and local notifications are used instead. If a user also decides against local notifications, a manual update of the test results is also possible.
 
-If a user did not receive a teleTAN from the health authority and/or has lost the QR code, a teleTAN needs to be retrieved from a hotline, which needs to ensure that users are permitted to perform an upload before issuing the teleTAN. It is then used as described before, starting from *Figure 4*, step 7. 
+If a user did not receive a teleTAN from the health authority and/or has lost the QR code, a teleTAN needs to be retrieved from a hotline. The hotline ensures that users are permitted to perform an upload before issuing the teleTAN. It is then used as described before, starting from *Figure 4*, step 7.
```
[the remark this answers is not in the document]
Comment (SabineLoss) [reply solution_architecture.md:92]: Yes, but here it is more generic about the tasks of the hotline, in my opinion.

[the remark this answers is not in the document]
Comment (OlMue) [reply solution_architecture.md:92]: Okay, good point, "users" is fine then!
(by the way, I wanted to say "a user", of course...)

**Comment by SabineLoss** [comment]
Added changes proposed by Oliver, fixed the typos found by Sebastian and resolved the conflict, merging now.

**Pull request merged**